scripts/xe-restore-metadata

```diff
@@ -100,6 +107,13 @@ test_sr "${sr_uuid}"
 
 sr_name=$(${XE} sr-param-get uuid=${sr_uuid} param-name=name-label)
```

Comment:
In xe-restore-metadata line 108:
sr_name=$(${XE} sr-param-get uuid=${sr_uuid} param-name=name-label)
^-----^ SC2034 (warning): sr_name appears unused. Verify use (or export if used externally).
                                  ^--------^ SC2086 (info): Double quote to prevent globbing and word splitting.

Did you mean:
sr_name=$(${XE} sr-param-get uuid="${sr_uuid}" param-name=name-label)